CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
-## 0.1.0 (Unreleased)
+## 1.0.0 (September 11, 2023)
+
+BREAKING CHANGES:
+- resource/service: Does not support `storageGB` anymore
```

Comment:
Maybe we could accept this field in the manifest, but issue a warning that this is a no-op? (Do not want to "break" existing users manifests)

Reply:
@ramonguiu

Reply:
I'm ok either way. We're moving from beta to GA and it's technically fine to not be backwards compatible.

I like the suggestion though. We could do this and in the warning state that this will be removed in the next minor release and that users should update their templates.

Reply:
sounds great